services/spd/trusty/trusty.c

```diff
@@ -354,7 +354,7 @@ static void trusty_cpu_resume(uint32_t on)
 
 static int32_t trusty_cpu_off_handler(u_register_t unused)
```

Comment:
Maybe you should rename this argument, now that it isn't unused. Same for the other functions in this patch.